packages/firestore/test/unit/specs/spec_builder.ts

```diff
@@ -458,11 +451,12 @@ export class SpecBuilder {
       if (!objUtils.contains(this.limboMapping, path)) {
         this.limboMapping[path] = this.limboIdGenerator.next();
       }
-      this.activeTargets[this.limboMapping[path]] = {
-        query: SpecBuilder.queryToSpec(Query.atPath(key.path)),
-        // Limbo doc queries are currently always without resume token
-        resumeToken: ''
-      };
+      // Limbo doc queries are currently always without resume token
+      this.addQueryToActiveTargetsIfNotYet(
```

Comment:
Limbo queries are single document queries that request the latest state of a document. Since we know that the document is out of sync, it probably would not make sense to use resume tokens.

Reply:
I thought this is what the code does here? I did not change this..